src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -63,85 +93,250 @@ private void CheckForAndConvertJsonConfig()
+            //                                       "\n - Original - Default .NET stack trace format."),
+            //     Options.StackTraceMode);
+
+            EditorGUILayout.Space();
+            EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
+            EditorGUILayout.Space();
 
-            GUILayout.Label(new GUIContent(GUIContent.none), EditorStyles.boldLabel);
+            Options.SendDefaultPii = EditorGUILayout.BeginToggleGroup(
+                new GUIContent("Send default Pii", "Whether to include default Personal Identifiable " +
+                                                   "Information."),
+                Options.SendDefaultPii);
+
+            Options.IsEnvironmentUser = EditorGUILayout.Toggle(
+                new GUIContent("Auto Set UserName", "Whether to report the 'Environment.UserName' as " +
+                                                    "the User affected in the event. Should be disabled for " +
+                                                    "Android and iOS."),
+                Options.IsEnvironmentUser);
+
+            Options.ServerNameOverride = EditorGUILayout.TextField(
+                new GUIContent("Server Name Override", "The name of the server running the application." +
+                                                       "\nThis option is an override."),
+                Options.ServerNameOverride);
+
+            EditorGUILayout.EndToggleGroup();
+
+            EditorGUILayout.Space();
+            EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
+            EditorGUILayout.Space();
+
+            Options.MaxBreadcrumbs = EditorGUILayout.IntField(
+                new GUIContent("Max Breadcrumbs", "Maximum number of breadcrumbs that get captured." +
+                                                  "\nDefault: 100"),
+                Options.MaxBreadcrumbs);
+
+            Options.ReportAssembliesMode = (ReportAssembliesMode)EditorGUILayout.EnumPopup(
+                new GUIContent("Report Assemblies Mode", "Whether or not to include referenced assemblies " +
+                                                         "in each event sent to sentry."),
+                Options.ReportAssembliesMode);
+        }
+
+        private void DisplayTransport()
+        {
+            Options.EnableOfflineCaching = EditorGUILayout.BeginToggleGroup(
+                new GUIContent("Enable Offline Caching", ""),
+                Options.EnableOfflineCaching);
+            Options.MaxCacheItems = EditorGUILayout.IntField(
+                new GUIContent("Max Cache Items", "The maximum number of items to keep in cache. SDK " +
+                                                  "deletes the oldest and migrates the sessions to the next to " +
+                                                  "maintain the integrity of your release health stats.\nDefault: 30"),
```

Comment:
Not sure it's worth talking about migrate session, it's too much internal stuff for the user at this point.
```suggestion
                new GUIContent("Max Cache Items", "The maximum number of files to keep in the disk cache. SDK " +
                                                  "deletes the oldest when limit is reached.\nDefault: 30"),
```